src/sentry/models/tagvalue.py

```diff
@@ -49,35 +47,6 @@ def is_valid_value(cls, value):
 
     def get_label(self):
         # HACK(dcramer): quick and dirty way to hack in better display states
-        if self.key == 'sentry:user':
-            return self.data.get('email') or self.value
-        elif self.key == 'sentry:function':
-            return '%s in %s' % (self.data['function'], self.data['filename'])
-        elif self.key == 'sentry:filename':
-            return self.data['filename']
-        elif self.key == 'sentry:release' and len(self.value) == 40:
+        if self.key == 'sentry:release' and len(self.value) == 40:
             return self.value[:12]
         return self.value
-
-    def get_absolute_url(self):
```

Comment:
I guess this is fine? I am not sure I can quickly figure out where this is used since this typically goes through an indirection that is hard to grep for.